docker-traceability-plugin/src/main/java/org/jenkinsci/plugins/docker/traceability/JenkinsInstance.java

```diff
@@ -0,0 +1,29 @@
+package org.jenkinsci.plugins.docker.traceability;
+
+import javax.annotation.CheckForNull;
+
+import jenkins.model.Jenkins;
+
+public class JenkinsInstance {
+
+    /**
+     * Jenkins.getInstance() is annotated with CheckForNull since 1.565.1
+     * CheckForNull was replaced with NonNull since 2.4
+     * 
+     * So, until the baseline is updated to 2.4, there is no way to make the build
+     * pass for all baselines due to FindBugs errors (unless ignored), i.e:
+     *      - Cannot remove the null check because it fails with baselines 1.565.1 - 2.3
+     *      - Cannot use the null check because it fails with baselines 2.4++
+     *
+     * This methods wraps the call to Jenkins.getInstance() to force CheckForNull, so that the
+     * same approach can be used with different versions.
+     * 
+     * TODO: Remove this class when baseline is bumped to 2.4 or higher.
+     * 
+     * @return a Jenkins instance
+     */
+    @CheckForNull
+    public static Jenkins get() {
+        return Jenkins.getInstance();
+    }
+}
```

Comment:
This is kind of hacky and I don't like it, but it was the alternative I could think of without polluting all methods using `Jenkins.getInstance()` with `@SuppressFBWarnings` which would increase the technical debt even more than this IMHO.

I did not consider ignoring FindBugs errors at all, I think the check should be performed and this is no reason to disable it with `<findbugs.failOnError>false</findbugs.failOnError>`